README.md

```diff
@@ -1,86 +1,135 @@
 
-1. Download and install the [latest version of DocFX](https://github.com/dotnet/docfx/releases). 
-1. Copy the repository to your computer and checkout the branch corresponding to the version of DevExpress controls your application uses. Do not use the **master** branch because it represents the version currently under development.
-    ```
-    git clone https://github.com/DevExpress/dotnet-eud.git
-    git switch 20.1
-    ```
-    If you do not have [Git](https://git-scm.com/) installed, use the GitHub web interface to select the branch and then download and extract the ZIP archive.
-  
-    ![Download ZIP](https://user-images.githubusercontent.com/20167812/29712204-4ffaee9e-89a1-11e7-8a0e-3ff0464adda4.png)
-1. You can make the following changes to the documentation, or skip this step if you want to reuse the end-user documentation as is:
-   
-   - remove unnecessary files;
-   - add new documents to your application;
-   - change screenshots to match your app's UI;
-   - create a [custom DocFX template](https://dotnet.github.io/docfx/tutorial/howto_create_custom_template.html).
-   
-   > You should update the *toc.yml* ([table-of-content](https://dotnet.github.io/docfx/tutorial/intro_toc.html)) files if you added or removed topics.
-1. Open a console window, change the current directory to the repository root folder and call the DocFX executable with the following parameters: *build* and the path to *docfx.json* in the downloaded _dotnet-eud_ repository. DocFX will place the generated documentation content into the *\_site* folder. Add the `--serve` switch to preview the generated website at http://localhost:8080 once the build process is complete. 
-    ```
-    docfx.exe build ../dotnet-eud/docfx.json --serve
-    ```
-1. Deploy the created documentation to a web server or browse the documentation directly from local file system. Since DocFX creates static HTML files only, no additional deployment or configuration steps are required.
+```bash
+dotnet tool update -g docfx
+```
 
-## Build Printer-Friendly PDF Files
-If your end users require a printed version, you can build a PDF file like this:
+> You can also download the latest version of DocFX directly: [releases](https://github.com/dotnet/docfx/releases).
 
-1. Ensure that you can successfully [build a website with DocFX](#build-your-own-documentation-website).
-1. Download and install [wkhtmltopdf](https://wkhtmltopdf.org/downloads.html).
-1. In the folder with `docfx.exe`, open a console window and add the `wkhtmltopdf` executable path to the `%PATH%` environment variable:
+Copy the repository to your computer and checkout the branch corresponding to the version of DevExpress controls your application uses. 
 
-    ```
-    set PATH=%PATH%;C:\Program Files\wkhtmltopdf\bin
-    ```
+Do not use the `master` branch because it is the version currently under development:
```

Comment:
I feel like we should move this line after the snippet (can't make the suggestion in the pr).